pkg/storaged/fsys-tab.jsx

```diff
@@ -0,0 +1,227 @@
+                return block.RemoveConfigurationItem(old_config, {});
+            else if (old_config && new_config && (new_dir != old_dir || new_opts != old_opts))
+                return block.UpdateConfigurationItem(old_config, new_config, {});
+        }
+
+        function mounting_dialog () {
+            dialog.open({ Title: _("Filesystem Mounting"),
+                          Fields: [
+                              { SelectOne: "mounting",
+                                Title: _("Mounting"),
+                                Options: [
+                                    { value: "default", Title: _("Default"), selected: !old_config },
+                                    { value: "custom", Title: _("Custom"), selected: !!old_config }
+                                ],
+                              },
+                              { TextInput: "mount_point",
+                                Title: _("Mount Point"),
+                                Value: old_dir,
+                                visible: function (vals) {
+                                    return vals.mounting == "custom";
+                                }
+                              },
+                              { TextInput: "mount_options",
+                                Title: _("Mount Options"),
+                                Value: old_opts,
+                                visible: function (vals) {
+                                    return vals.mounting == "custom";
+                                }
+                              }
+                          ],
+                          Action: {
+                              Title: _("Apply"),
+                              action: function (vals) {
+                                  return maybe_update_config(vals.mounting == "custom",
+                                                             vals.mount_point, vals.mount_options);
+                              }
+                          }
+            });
+        }
+
+        function mount () {
+            return block_fsys.Mount({});
+        }
+
+        function unmount () {
+            return block_fsys.Unmount({});
+        }
+
+        // See format-dialog.jsx for why we don't offer editing
+        // fstab for the old UDisks2
+
+        return (
+            <div>
+                <div className="tab-actions">
+                    <FormatButton client={this.props.client} block={this.props.block}/>
+                </div>
+                <table className="info-table-ct">
+                    <tr>
+                        <td>{_("Name")}</td>
+                        <td>
+                            <StorageLink onClick={rename_dialog}>
+                                {this.props.block.IdLabel || "-"}
+                            </StorageLink>
+                        </td>
+                    </tr>
+                    { !self.props.client.is_old_udisks2 ?
+                    <tr>
```

Comment:
According to the [coding guidelines](https://github.com/cockpit-project/cockpit/wiki/Cockpit-Coding-Guidelines#javascript-style), we want to wrap multiline jsx in parentheses. Now, this is a case of nested jsx...
In any case, I think the indentation needs to change - according to [Crockford](http://javascript.crockford.com/code.html) because of the operator `?` and since this is inside `{ }`.